docs/theming.md

```diff
@@ -98,7 +98,7 @@ MDCContainerScheme *containerScheme = [[MDCContainerScheme alloc] init];
 containerScheme.colorScheme.primaryColor = [UIColor redColor];
 
 // Or assign a customized scheme instance:
-let shapeScheme = MDCShapeScheme()
```

Comment:
I can't comment but should we also update the import statement? If you want we can do it in a follow up.

Reply:
I don't know about the import statement. I saw Swift in Obj-C snippet so I fixed it. Feel free to open a PR with an update to the import statement as you see fit.